chat.js

```diff
@@ -563,12 +563,6 @@ class CommandContext {
 	errorReply(message) {
 		this.sendReply(`|error|` + message.replace(/\n/g, `\n|error|`));
 	}
-	/**
-	 * @param {string} html
-	 */
-	addBox(html) {
-		this.add(`|html|<div class="infobox">${html}</div>`);
-	}
```

Comment:
Would rather not remove this.

Reply:
(It matches `sendReplyBox`, so it's useful as API surface even if nothing uses it currently.)

Reply:
Sure, makes sense. Should I add something like <code>addModBox</code> to match or just keep that one inline?

Reply:
Either is fine by me.